### ISSUE TITLE
NPM list reports errors

### ISSUE THREAD

**Issue by nicoduj** [lopelex/harmony-websocket#9]
Hi, I have some unmet dependencies on some installations when I do a npm -list, I think that latest is not handled correctly if the dependency (very common for request) is referenced by another packet in npm directory. Som users are exepriencing strnage behaviors sometimes with network issues, and reinstalling seems to fix it most of the time. I am not really used to @latest syntax in package.json,  do you have any idea if it could be a problem at all ? 

```bash
+-- homebridge-harmony@1.3.3
| +-- harmony-websocket@1.5.0
| | +-- UNMET DEPENDENCY request@latest
| | +-- request-promise@4.2.5
| | | +-- bluebird@3.7.2
| | | +-- request-promise-core@1.1.3
| | | | `-- lodash@4.17.15
| | | +-- stealthy-require@1.1.1
| | | `-- tough-cookie@2.5.0 deduped
| | +-- UNMET DEPENDENCY websocket@latest
| | `-- UNMET DEPENDENCY websocket-as-promised@latest
| +-- harmonyhubjs-discover@1.1.1
| | `-- debug@2.6.9
| |   `-- ms@2.0.0
| +-- request@2.88.2 invalid
| | +-- aws-sign2@0.7.0 extraneous
| | +-- aws4@1.9.1 extraneous
| | +-- caseless@0.12.0 extraneous
| | +-- combined-stream@1.0.8 extraneous
| | +-- extend@3.0.2 extraneous
| | +-- forever-agent@0.6.1 extraneous
| | +-- form-data@2.3.3 extraneous
| | +-- har-validator@5.1.3 extraneous
| | +-- http-signature@1.2.0 extraneous
| | +-- is-typedarray@1.0.0 extraneous
| | +-- isstream@0.1.2 extraneous
| | +-- json-stringify-safe@5.0.1 extraneous
| | +-- mime-types@2.1.26 extraneous
| | +-- oauth-sign@0.9.0 extraneous
| | +-- performance-now@2.1.0 extraneous
| | +-- qs@6.5.2 extraneous
| | +-- safe-buffer@5.2.0 extraneous
| | +-- tough-cookie@2.5.0
| | | +-- psl@1.8.0
| | | `-- punycode@2.1.1
| | +-- tunnel-agent@0.6.0 extraneous
| | `-- uuid@3.4.0 extraneous
| +-- websocket@1.0.31 extraneous
| `-- websocket-as-promised@1.0.1 extraneous
```

**Comment by lopelex** [comment]
It would probably be better to fix the versions.

**Comment by nicoduj** [comment]
Ok, do you want me to do a pull request ? I suggest we fixed in semver format to latest major version of each package, don't you think so ? Something like request : "^2.88.0"  for example ?

**Comment by lopelex** [comment]
Would appreciate a pull request.

**Comment by lopelex** [comment]
published your version 1.5.1
